wallet/src/de/schildbach/wallet/di/ConfigurationModule.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 Dash Core Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
```

Comment:
I wonder how many files have an Apache 2.0 License.

Reply:
This is the old template I used, will fix. Most of the DashPay files have Apache licenses.